sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -106,13 +108,14 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
                 throw;
             }
 
+            string statusCodeStr = message.Response.Status.ToString(CultureInfo.InvariantCulture);
```

Comment:
Why cache it if we don't use it in both branches?

Reply:
used on line 137 as well (effectively on both branches)